wandb/sdk/wandb_run.py

```diff
@@ -1953,7 +1980,9 @@ def _console_start(self) -> None:
 
         if self._use_redirect:
             # setup fake callback
+            # TODO: we dont need these redirect callbacks and just use the underlying methods
```

Comment:
should we address this todo in this PR?

Reply:
i can do that.. it should be safe enough.